### DISCUSSION TITLE
Single shard

### DISCUSSION THREAD

**Discussion by pietermartin** [citusdata/citus#5420] [General]
Hi,

We have many tables, 50 000+. We'd like to put certain schemas on separate workers/nodes, but have no need to shard individual tables across workers.

Does it make sense to use a shard count of 1?
Will the cluster be able to be balanced?

Will we instead need to move the shards individually using `citus_copy_shard_placement`?

Thanks
Pieter

**Comment by marcocitus** [comment]
The cluster will be balanced when calling rebalance_table_shards, but not when tables are created, so you may need to rebalance after creating new tables. Make sure to use colocate_with in create_distributed_table for tables in the same schema, and colocate_with :='none' for the first table in a new schema.

(wish GitHub showed some visual hint for new discussion topics)

**Reply by pietermartin** [reply]
Not quite following why there is a need for `rebalance_table_shards`. Seeing as its going to one table there is nothing to rebalance or am I missing something?

**Comment by onurctirtir** [comment]
There is a much easier way of distributing a PostgreSQL cluster based on schemas, as of 12.0, see https://www.citusdata.com/blog/2023/07/31/schema-based-sharding-comes-to-postgres-with-citus/.
